doc/node_module_versions.json

```
@@ -0,0 +1,55 @@
+[
```

```suggestion
[
  { "modules": 72, "runtime": "node",     "variant": "v8_7.4",               "versions": "12.0.0-pre" },
```

Refs: https://github.com/nodejs/node/pull/26685

---

/cc @targos

---

manually added, thanks